test/battle/ability/anger_shell.c

```diff
@@ -49,15 +49,15 @@ SINGLE_BATTLE_TEST("Anger Shell lowers Def/Sp.Def by 1 and raises Atk/Sp.Atk/Spd
         ANIMATION(ANIM_TYPE_MOVE, MOVE_TACKLE, opponent);
         ABILITY_POPUP(player, ABILITY_ANGER_SHELL);
         ANIMATION(ANIM_TYPE_GENERAL, B_ANIM_STATS_CHANGE, player);
-        MESSAGE("Wobbuffet's Defense fell!");
+        MESSAGE(SPECIES_NAME(SPECIES_WOBBUFFET) "'s Defense fell!");
```

Comment:
Honestly I would keep `Wobbuffet's`. Writing `SPECIES_NAME(SPECIES_WOBBUFFET) "` every time seems frustrating to do :D

Reply:
But can you not check multiply mons inside one message this way?

Reply:
Nope, which is why it got drafted. We bumped into an issue where preproc didn't take variables into account :(